src/app/app.page/app.page.component.html

```diff
@@ -1831,9 +1850,42 @@ <h4 class="card-title text-left" i18n="@@newLiveStreamCardTitle">
                                                                     rows="3">{{appSettings.remoteAllowedCIDR}}
                                                             </textarea>
                                                             </div>
+                                                        </div>
 
+                                                        <div class="col-xs-12" style="margin-top:12px">
+                                                            <div class="checkbox">
+                                                                <input [(ngModel)]="appSettings.jwtControlEnabled"
+                                                                   id="jwtFilter"
+                                                                   name="jwtFilter" type="checkbox">
+
+                                                                <label for="jwtFilter"
+                                                                   i18n="@@jwtFilter">
+                                                                Enable JWT Filter<a
```

Comment:
JWT filter for what? 
I think create a sub-menu for RESTful security and put these items there